app/views/candidate_interface/volunteering/role/_form.html.erb

```diff
@@ -25,8 +25,8 @@
       <div class="app-work-experience__start-date" data-qa="end-date">
         <%= f.govuk_date_field :end_date, omit_day: true, legend: { text: t('application_form.volunteering.end_date_restructured_work_history.label'), size: 'm' }, hint: { text: t('application_form.volunteering.end_date_restructured_work_history.hint_text') } %>
         <div class="govuk-form-group">
-          <%= f.hidden_field :end_date_unknown, value: false %>
-          <%= f.govuk_check_box :end_date_unknown, true, multiple: false, label: { text: t('application_form.volunteering.end_date_unknown_checkbox') } %>
+          <%= f.hidden_field :end_date_unknown, value: 'false' %>
```

Comment:
Doesn't the hidden field get rendered by `govuk_check_box` on the next line?

Reply:
Discussed offline